components/header.js

```diff
@@ -0,0 +1,72 @@
+import "./link-button/link-button.js";
+
+const template = document.createElement("template");
+template.innerHTML = `
+  <style>
+    :host {
+      position: fixed;
+      top: 0;
+      left: 0;
+      padding: 20px 0;
+      width: 100%;
+      border-bottom: 1px solid #D3D3D3;
+      background-color: rgba(255, 255, 255, 0.8);
+    }
+
+    header {
+      width: 80%;
+      margin: 0 auto;
+      justify-content: space-between;
+      display: flex;
+      align-items: center;
+    }
+
+    .buttons-container {
+      display: flex;
+    }
+
+    .logo {
+      display: flex;
+      gap: 10px;
+      align-items: center;
+      text-decoration: none;
+    }
+
+    .logo img {
+      width: 45px;
+      height: 22.5px;
+    }
+
+    .logo span {
+      font-size: 20px;
+      font-weight: 500;
+      color: black;
+    }
+
+    .logo img,
+    .logo span {
+      vertical-align: middle;
+    }
```

Comment:
이 컴포넌트에는 `a` 요소가 하나 밖에 없는데, 굳이 클래스 선택자를 사용할 이유가 없어 보입니다. 참고로 컴포넌트 기반으로 개발을 하면 이렇게 상대적으로 클래스를 적게 쓰고 되고 따라서 클래스 명명에 대해서 고민할 일도 적어지게 되죠. 소소한 장점입니다.

```suggestion
    a {
      display: flex;
      gap: 10px;
      align-items: center;
      text-decoration: none;
    }

    a img {
      width: 45px;
      height: 22.5px;
    }

    a span {
      font-size: 20px;
      font-weight: 500;
      color: black;
    }

    a img,
    a span {
      vertical-align: middle;
    }
```

Reply:
사실 저도 이 부분을 a로 할 지 logo로 할 지 고민하다가 class를 사용하면 각 요소의 역할을 구체적으로 명명하는 느낌이라 가독성?이 늘거라고 판단해 logo 클라스를 생성했는데 코멘트를 보고 생각해보니 덜 쓰는게 좋고 best practice 인 것 같네요ㅎㅎ 피드백 감사합니다!